draft-ietf-tls-tls13.md

```diff
@@ -2107,15 +2110,16 @@ to any second HelloRetryRequest which was sent in the same connection
 (i.e., where the ClientHello was itself in response to a
 HelloRetryRequest).
 
-Otherwise, the client MUST send a ClientHello with a new KeyShare
-extension to the server. The client MUST append a new KeyShareEntry
-list which is consistent with the "selected_group" field to the groups
-in its original KeyShare.
+Otherwise, the client MUST send a ClientHello with an updated KeyShare
+extension to the server, or a new extension if it was previously omitted.
+The client MUST append a new KeyShareEntry for the selected_group
+to the groups in its original KeyShare list.
```

Comment:
selected_group is a field so "for" seems weird. If you prefer "for" then you need to say "for the group indicated in selected group"